packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -511,47 +570,71 @@ async def folder_create(
         )
         if entry_exists:
             raise FolderAlreadyExistsError(
-                product_name=product_name, folder=name, parent=parent, gid=gid
+                product_name=product_name, folder=name, parent=parent, gids=gids
             )
 
+        # `permissions_gid` is compted as follows:
+        # - `folder has a parent?` taken from the resolved access rights of the parent folder
+        # - `is root folder, a.k.a. no parent?` taken from the user's primary group
+        permissions_gid: _GroupID | None = None
```

Comment:
as I said in my PR, there is no need to define the type of the variable, as both mypy and pylance automatically deduce it. but if that helps you ok.